R/data_set.R

```diff
@@ -167,12 +167,13 @@ setMethod("data_set", c("mrgmod", "missing"), function(x, object, ...) {
 #' If both lower and upper case versions of the name are present in the data 
 #' frame, no changes will be made. 
```

Comment:
Just a question, Kyle. Are the columns listed above the only data items will be controlled by `lctran()` or `uctran()`. If so, wouldn't be better to also include things like `ID`, `MDV`? 
Additionally, any specific reason for this: "If both lower and upper case versions of the name are present in the data frame, no changes will be made"

Reply:
Thanks, Shen. 

1. There is no MDV
2. ID is always capital (that was one of the motivations for this)
3. We can't have both `amt` and `AMT` in the data set and then turn `amt` into `AMT`; this really shouldn't happen and we wait until the data is submitted for simulation where this will turn into an error.

Reply:
Makes sense. Thanks Kyle!